hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -401,6 +406,50 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 			log.Info("hostedcluster is still deleting", "name", req.NamespacedName)
 			return ctrl.Result{RequeueAfter: clusterDeletionRequeueDuration}, nil
 		}
+
+		// This new condition is necessary for OCM people to report the cloud dangling objects to the user.
+		// The grace period is customizable using the flag --destroy-grace-period it's a time.Duration field type.
+		// before remove the finalizer on the HostedCluster. This condition only will be created when !SelfManage and
```

Comment:
Was there text that was supposed to be before the `before`? I'm not sure I understand what this sentence fragment is saying.

Reply:
Yep, that was old.